dialoptions.go

```diff
@@ -680,8 +681,8 @@ func WithResolvers(rs ...resolver.Builder) DialOption {
 // channel will exit idle mode when the Connect() method is called or when an
 // RPC is initiated.
 //
-// By default this feature is disabled, which can also be explicitly configured
-// by passing zero to this function.
+// A default timeout of 30 min will be used if this dial option is not set at
```

Comment:
Nit: "minutes" instead of "min" would be more formal / appropriate for godoc comments.